CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 Changes to Calva.
 
 ## [Unreleased]
+- [Add debugging](https://github.com/BetterThanTomorrow/calva/issues/469)
+- Fix so that Calva treats symbol containing the quote character correctly.
```

Comment:
What's this about? (The part about quote characters.)

Reply:
This was added by you a while back, and I guessed changed or removed since then. Likely was a change you merged in and maybe referenced differently in this doc. I'll remove it.
![image](https://user-images.githubusercontent.com/12722744/77950193-01820c80-727d-11ea-909a-9a559a40bc9e.png)

Reply:
I think it may be related to this: [Fix: Var quoted symbols are treated as reader tags](https://github.com/BetterThanTomorrow/calva/issues/584)